core/src/main/scala/algebra/Order.scala

```diff
@@ -148,6 +163,27 @@ object Order extends OrderFunctions {
    */
   implicit def ordering[A](implicit ev: Order[A]): Ordering[A] =
     new Ordering[A] {
-      def compare(x: A, y: A) = ev.compare(x, y)
+      def compare(x: A, y: A): Int = ev.compare(x, y)
+    }
+
+  /**
+   * A `Monoid[Order[A]]` can be generated for all `A` with the following
+   * properties:
+   *
+   * `empty` returns a trivial `Order[A]` which considers all `A` instances to
+   * be equal.
+   *
+   * `combine(x: Order[A], y: Order[A])` creates an `Order[A]` that first
+   * orders by `x` and then (if two elements are equal) falls back to `y`.
+   *
+   * @see [[Order.andThen]]
+   */
+  implicit def orderMonoid[A]: Monoid[Order[A]] =
+    new Monoid[Order[A]] {
+      val empty: Order[A] = new Order[A] {
```

Comment:
I think this could be lifted out to `Order.allEqual` or something. This could be useful when people need an order, but don't care, and don't want to do the roundabout of `Monoid.empty[Order[A]]` which is strange thing to read when the rest of the code might have nothing to do with Monoids.